service/src/main/kotlin/fi/espoo/evaka/reservations/Reservations.kt

```diff
@@ -517,3 +536,51 @@ private fun Database.Transaction.insertReservation(
         }
         .exactlyOne<AttendanceReservationId>()
 }
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+sealed interface UsedService {
```

Comment:
This sounds quite a lot like a service bean 😅 Might consider renaming to e.g. UsedServiceTimes to reduce initial confusion

Reply:
Can be done later if needed :+1: